actions/exterminate

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python3
+
+# Copyright 2018 Canonical Ltd.
+# Licensed under the AGPLv3, see LICENCE file for details.
+
+# Load modules from $JUJU_CHARM_DIR/lib.
+import sys
+sys.path.append('lib')
```

Comment:
Is this going to work? This is a relative path, so I guess it's relying on juju executing the action from the correct directory?

Reply:
Yes, it's relying on that.